app/frontend/src/components/forms/manage/TeamManagement.vue

```diff
@@ -57,15 +57,39 @@
       no-data-text="Failed to load team role data"
       :search="search"
     >
+      <!-- custom header markup - add tooltip to roles -->
+      <template v-for="h in headers.slice(2)" v-slot:[`header.${h.value}`]="{ headers }">
+        <v-tooltip
+          :key="h.value"
+          bottom>
+          <template v-slot:activator="{ on }">
+            <span v-on="on">{{h.text}}</span>
+          </template>
+          <span>{{h.description}}</span>
+        </v-tooltip>
+      </template>
```

Comment:
Style: Formatting here looks a bit wonky. Lines 62-64 can be all as one line assuming VSCode Format Document allows it.
Also generally it's better to have a space between the double curly braces and the internal contents for readability:
`{{ h.description }}`